<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SenderWindow</class>
 <widget class="QWidget" name="SenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MavLink Sender</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="16" column="0">
    <widget class="QPushButton" name="stop">
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="remoteIPLine"/>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="remotePortLabel">
     <property name="text">
      <string>Remote Port:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="remoteIPlabel">
     <property name="text">
      <string>Remote IP: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="compIDlabel">
     <property name="text">
      <string>CompID: </string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="start">
     <property name="text">
      <string>start</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="4">
    <widget class="QLabel" name="locPosLabel">
     <property name="text">
      <string>Local Position:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="droneNameLabel">
     <property name="text">
      <string>DroneName</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="sysIDLine"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="remotePortLine"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="compIDLine"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="SysIDLabel">
     <property name="text">
      <string>SystemID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="MotionCaptureLabel">
     <property name="text">
      <string>Motion Capture for: </string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QSlider" name="rateSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLabel" name="Ratelabel">
     <property name="text">
      <string>Rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>remotePortLabel</zorder>
  <zorder>remotePortLine</zorder>
  <zorder>MotionCaptureLabel</zorder>
  <zorder>remoteIPlabel</zorder>
  <zorder>remoteIPLine</zorder>
  <zorder>droneNameLabel</zorder>
  <zorder>stop</zorder>
  <zorder>locPosLabel</zorder>
  <zorder>SysIDLabel</zorder>
  <zorder>sysIDLine</zorder>
  <zorder>compIDlabel</zorder>
  <zorder>compIDLine</zorder>
  <zorder>start</zorder>
  <zorder>rateSlider</zorder>
  <zorder>Ratelabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
